gen3/bin/kube-setup-ohdsi.sh

```diff
@@ -87,6 +87,7 @@ setup_secrets() {
     export DB_HOST=$(jq -r ".db_host" <<< "$dbcreds")
 
     export FENCE_URL="https://${hostname}/user/user"
+    export ARBORIST_URL="https://${hostname}/auth/mapping"
```

Comment:
This will be used for inside-Kubernetes communication, right? Because if this is the case, I'd suggest to use `http://arborist-service.default.svc.cluster.local/auth/mapping` or take it from the manifest.